frontend/javascripts/oxalis/model/volumetracing/volumelayer.js

```diff
@@ -193,6 +239,13 @@ class VolumeLayer {
     }
     const maxCoord2d = this.get2DCoordinate(this.maxCoord);
 
+    // The maximum area is scaled by 3 as the min and maxCoord will always be three slices apart,
+    // because in lines 171 + 172 a value of 2 is subtracted / added when the values get updated.
```

Comment:
I would rather reference the location by function name, the line number reference is very brittle.